### ISSUE TITLE
Can't access nested iframe contents

### ISSUE THREAD

**Issue by Jip-Hop** [nwjs/nw.js#7257]
NWJS Version : 0.43.0-beta1sdk
Operating System : MacOS 10.14.6

### Expected behavior

Should be able to access Cross Origin iframe contents in nested iframes, from a frame with access to Node.js APIs. This worked before in v0.42.6.

### Actual behavior

<img width="1440" alt="Screenshot 2019-12-04 at 16 40 06" src="https://user-images.githubusercontent.com/2871973/70157967-bed96a80-16b6-11ea-8ccb-8208af2ce75c.png">

```
Uncaught DOMException: Blocked a frame with origin "chrome-extension://jpignaibiiemhngfjkcpokkamffknabf" from accessing a cross-origin frame.
    at accessSubFrameContents (chrome-extension://jpignaibiiemhngfjkcpokkamffknabf/index.html:35:45)
```

### How to reproduce

See the example project in [Archive.zip](https://github.com/nwjs/nw.js/files/3922752/Archive.zip)

**Comment by rogerwang** [comment]
It works for me with this build: http://dl.nwjs.io/live-build/nw43/20191207-162000/0181cca2f/v0.43.0/

**Comment by rogerwang** [comment]
close until more information is available.

**Comment by Jip-Hop** [comment]
Thanks, it's fixed in 0.43.0!
